src/app/shared-services/staypoint/staypoint.service.ts

```diff
@@ -15,9 +15,9 @@ import { take } from 'rxjs/operators'
 })
 export class StaypointService {
   // for meaning of these two parameters, please see detectStayPoints() documentation
-  // if you change one or both, please also update the associated detected staypoints in staypoint.service.spec.ts
-  readonly DIST_THRESH_METERS = 100
-  readonly TIME_THRESH_MINUTES = 15
+  // if you change one or both, please also update the associated detected staypoints in staypoint.service.spec.fixtures.ts
+  static DIST_THRESH_METERS = 100
+  static TIME_THRESH_MINUTES = 15
```

Comment:
If I'm not mistaken, the `readonly` can actually sit alongside `static`; no need to remove it ;)